src/prop/minisat/minisat.h

```diff
@@ -107,6 +107,9 @@ class MinisatSatSolver : public CDCLTSatSolver, protected EnvObj
   /** Retrieve the refutation proof of this SAT solver. */
   std::shared_ptr<ProofNode> getProof() override;
 
+  /** Get proof sketch, not used */
```

Comment:
Maybe do not add `not used` in the comments, it's prone to be left over when this may be changed in the future (even if this is unlikely)?